### PR TITLE
Fix/dynamic apps

### PR DESCRIPTION
**Pull request by xquanluu**
https://github.com/jambonz/jambonz-feature-server/issues/498

### CODE REVIEW

File: lib/apps-map.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        {
+          type: 'user',
+          name: ''
+        }
+      ]
+    }]
+  }
+};
+
+const utilizeJambonzApp = (type, {account_sid, name, caller_id}) => {
```
Comment (davehorton) [inline lib/apps-map.js:36]: can you rename to createJambonzApp

File: lib/middleware.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const listTaskNames = require('./utils/summarize-tasks');
 const {
   JAMBONES_MYSQL_REFRESH_TTL
 } = require('./config');
+const { utilizeJambonzApp } = require('./apps-map');
```
Comment (davehorton) [inline lib/middleware.js:16]: can you change the file name from apps-map to dynamic-apps

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/apps-map.js (hunk 1 above)
Comment (xquanluu) [reply lib/apps-map.js]: fixed

File: lib/middleware.js (hunk 2 above)
Comment (xquanluu) [reply lib/middleware.js]: fixed

**Review by davehorton — APPROVED** [review]

**Pull request merged**